src/components/routes/Host/ListingForm/AsideContent.tsx

```diff
@@ -0,0 +1,213 @@
+import * as React from 'react';
+import ListItem from 'shared/ListItem';
+import Svg from 'shared/Svg';
+import LazyImage from 'shared/LazyImage';
+
+interface AsideContentInterface {
+  [name: string]: React.ReactNode;
+}
+
+const HomeType = () => (
+  <div>
+    <h3>Type of Home</h3>
+    <br />
+    <h4><strong>Entire place</strong></h4>
+    <p>The whole space is dedicated to guests. This may be a house, apartment, condo, loft, etc.</p>
+    <br />
+    <h4><strong>Private room</strong></h4>
+    <p>A single room within a place is dedicated to guests. While other areas could be shared, guests have their own private room for sleeping.</p>
+    <br />
+    <h4><strong>Shared room</strong></h4>
+    <p>Bedrooms or a common area that could be shared with others are dedicated to guests.</p>
+  </div>
+);
+
+const Title = () => (
+  <div>
+    <h3>Listing Name</h3>
+    <br />
+    <p>&quot;Beautiful Home with Wifi and Views&quot;</p>
+  </div>
+);
+
+const Description = () => (
+  <div>
+    <h3>Listing Description</h3>
+    <br />
+    <p>Include a summary of your place to help guests imagine what it will be like to stay there. Note what makes your place special, mention what the neighborhood is like, and add anything else guests should know before arriving.</p>
+  </div>
+)
+
+const FullAddress = () => (
+  <div>
+    <h3>Full Address</h3>
+    <br />
+    <p>When adding the Full Address -- don’t forget to add a zip code!</p>
+  </div>
+);
+
+const ListingPicUrl = () => (
+  <div>
+    <h3>Cover Photo</h3>
+    <br />
+    <p>High-quality photos are a must as our guests are typically business travelers. Ideal image dimensions are 1440x960. The maximum file size is 5 MB and the file types we accept are .jpg &amp; .png.</p>
+    <br />
+    <h4>Examples:</h4>
+    <br />
+    <div className="images-container">
+      <LazyImage className="image-container--horizontal" src="https://s3-us-west-2.amazonaws.com/beenest-public/images/photo-examples/Do_LA_Bedroom.jpg" />
+      <LazyImage className="image-container--horizontal" src="https://s3-us-west-2.amazonaws.com/beenest-public/images/photo-examples/Do_LA_Bedroom_2.jpg" />
+      <LazyImage className="image-container--vertical" src="https://s3-us-west-2.amazonaws.com/beenest-public/images/photo-examples/bad_cover_example.png" />
+    </div>
+  </div>
+);
+
+const Photos = () => (
+  <div>
+    <h3>Listing Photos</h3>
+    <br />
+    <p>Beenest recommends listings have at least 5 high-definition photos available. Photos are the best way to show off your place to potential guests. <strong>Pro tip:</strong> provide multiple well-lit photos with different angles of bedrooms, bathrooms, and public spaces so guests know exactly what to expect.</p>
```

Comment:
i would use our typography instead of the native html tags like `strong` and `i` and things like that. we can control the font-size/line height to our styles instead of letting the browser handle  it.  I would say the same with `<br />`, try and keep it in the css so it's consolidated in one place and only use `<br />` when you really have to

Reply:
sounds good. removed `<br />` and added appropriate margins instead. Using `<span>` in order to bold words. (there are no italics in any content _yet_).